opensrp-app/src/main/java/org/smartregister/service/HTTPAgent.java

```diff
@@ -130,6 +130,7 @@ private HttpURLConnection initializeHttp(String requestURLPath, boolean useBasic
         try {
             urlConnection = initializeHttp(postURLPath, true);
 
+            urlConnection.setDoOutput(true);
```

Comment:
what is the reason for this change. 

This flag may not be required for GET requests

Reply:
This is in a POST request. A test was failing due to this missing

Reply:
I see i thought this was on `initializeHttp` method